install.sh

```diff
@@ -141,13 +141,6 @@ is_running_internal_install() {
 ############################################################
 # Set up variables based on command line arguments.
 #
-# Sets ${DRY_RUN} for --dry-run
```

Comment:
Why did these go away btw?

Reply:
Ohh, thx. Good question...

Reply:
Ahh because we would need to list all of them which is tediuous

Reply:
I though ahh comon drop it, its parsing args of course we set the variables and we can see it :-)=

Reply:
OK, I'm not too fussed either way, so leave it if you changed it already.